packages/rainbow-line/src/App.tsx

```diff
@@ -86,30 +90,80 @@ function App() {
     [filters],
   );
 
-  const getTargetData = useCallback(() => {
-    if (filters["parkingLot"] === "all") {
-      return [...processedDataLot1, ...processedDataLot2].reduce(
-        (result, current, index, parent) =>
-          reducePlacement(
-            filters["parkingLot"] as
-              | Exclude<Placement, "fukui-station-east-entrance" | "tojinbo-shotaro">
-              | "all",
-            [result as AggregatedData[], current, index, parent],
-          ),
-        [] as RainbowLineAggregatedData[],
-      );
-    } else if (filters["parkingLot"] === "rainbow-line-parking-lot-1-gate") {
-      return processedDataLot1.map((row) => ({
-        ...row,
-        [`${filters["parkingLot"]} total count`]: row["total count"],
-      }));
-    } else {
-      return processedDataLot2.map((row) => ({
+  const aggregatedParkingLotData = useCallback(
```

Comment:
```suggestion
  const aggregateParkingLotData = useCallback(
```

関数なので過去形の命名は違和感があります